article/app/views/fragments/articleBodyGarnett.scala.html

```diff
@@ -79,6 +79,10 @@
 
                             @BodyCleaner(article, amp = amp)
 
+                            @if(article.tags.isFootball && !article.fields.sensitive){
```

Comment:
It doesn't have to be an A/B test to have a switch - we could check the value of a switch here